src/Powercord/plugins/pc-guilds/components/Guild.jsx

```diff
@@ -146,7 +146,7 @@ const Guild = class Guild extends React.Component {
                   {this.props.guild.icon
                     ? <img
                       className={guildClasses.icon} alt='Server Icon' width='48' height='48'
-                      src={`https://cdn.discordapp.com/icons/${this.props.guild.id}/${this.props.guild.icon}.png`
+                      src={`https://cdn.discordapp.com/icons/${this.props.guild.id}/${this.props.guild.icon}.` + (this.state.hovered && this.props.guild.icon.startsWith("a_") ? "gif" : "png")
```

Comment:
Did you know that using + in a template string AND with big gross fat ugly `"`s is 106% illegal and is a bannable offense?!!1!
patch that before eslint eats ur ass :^)
```suggestion
                      src={`https://cdn.discordapp.com/icons/${this.props.guild.id}/${this.props.guild.icon}.${this.state.hovered && this.props.guild.icon.startsWith("a_") ? 'gif' : 'png'}`
```